src/tests/unit/tests/electron/platform/android/scan-results-to-unified-rules.test.ts

```diff
@@ -11,7 +11,7 @@ import { RuleResultsData, ScanResults } from 'electron/platform/android/scan-res
 import { convertScanResultsToUnifiedRules } from 'electron/platform/android/scan-results-to-unified-rules';
 import { buildRuleInformation, buildRuleResultObject, buildScanResultsObject } from './scan-results-helpers';
 
-describe('ScanResultsToUnifiedResults', () => {
+describe('ScanResultsToUnifiedRules', () => {
```

Comment:
describe was using the wrong class, meaning that the snapshots were also using the wrong name